### DISCUSSION TITLE
Schematic world map

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#372] [Q&A] [answered]
For better orientation, it would be helpful to have a rough, schematic world map.
I'm currently planning something in South Africa/Namibia. Nothing is currently displayed between Europe and Southern Africa.
A world map that only shows the borders of countries would be sufficient.
Or at least the outlines of the continents...

**Comment by devemux86** [comment]
@Leeb64 thanks for the suggestion.

There is a world.map that can be manually opened with other offline maps:
https://download.mapsforge.org/maps/v5/world/

But we need to improve it. This is also discussed in:
- https://github.com/devemux86/cruiser/discussions/101

**Reply by Leeb64** [reply]
That's OK!
Could be implemented!(Until something better is available)

**Reply by kds112** [reply]
It could be an option in the map settings. If set, the world maps is downloaded in the background 👍

**Comment by devemux86** [comment, marked as answer]
Let's continue the discussion for this feature in the older existing topic:
- https://github.com/devemux86/cruiser/discussions/101
